astropy/coordinates/angle_utilities.py

```diff
@@ -636,7 +636,7 @@ def sexagesimal_to_string(values, precision=None, pad=False, sep=(':',),
         literal.append('{last_value}{sep[2]}')
     literal = ''.join(literal)
     return literal.format(np.copysign(values[0], sign),
-                          int(values[1]), values[2],
+                          int(np.nan_to_num(values[1])), values[2],
```

Comment:
Thanks for the PR!

I don't think this is a good solution, though, as it now gives a misleading answer. Please see https://github.com/astropy/astropy/issues/11473#issuecomment-813051722 for my suggested fix.

Also, this will definitely need a test!